dist/js/drop.js

```diff
@@ -421,7 +422,8 @@ function createContext() {
         }
 
         if (!this.drop.parentNode) {
```

Comment:
Hi,

I experience exception from this line, that `this.drop` is undefined.
Just wondering how would this happens?

Thanks!